### ISSUE TITLE
大佬，为啥我使用21.3.18 这个版本的 efi ，上big sur 没法 扩展屏幕了

### ISSUE THREAD

**Issue by mxm646690173** [Pinming/Dell-Inspiron-7590-Hackintosh-Opencore#103]
机型： 灵越7590   屏幕 1080p

efi版本：21.3.18

系统版本：11.2.3

问题是：使用了您的 efi 之后，无法点亮扩展屏幕，只能点亮笔记本自带的屏幕，之前使用 10.15.7 的时候能点亮扩展屏幕，雷电接口功能正常

**Comment by MongQi** [comment]
我也出现了同样的问题，HDMI和随航都不能用了，目前还不知道原因，型号是7591 1080p 和你一模一样的版本

**Comment by MongQi** [comment]
昨天尝试了tctien342的180421的引导后，发现Type-C转HDMI已经恢复正常了，但是原生HDMI还是闪屏，而且每次插上只能显示1秒就黑屏

**Comment by mxm646690173** [comment]
> 昨天尝试了tctien342的180421的引导后，发现Type-C转HDMI已经恢复正常了，但是原生HDMI还是闪屏，而且每次插上只能显示1秒就黑屏

是的,tctien342 的版本, 进入系统之后,建议你睡眠大概 1 分钟,再唤醒,就不会黑屏了,直到你重启,这个,tctien342 的文档里面有说的很清楚,你可以看下的

**Comment by MongQi** [comment]
> > 昨天尝试了tctien342的180421的引导后，发现Type-C转HDMI已经恢复正常了，但是原生HDMI还是闪屏，而且每次插上只能显示1秒就黑屏
> 
> 是的,tctien342 的版本, 进入系统之后,建议你睡眠大概 1 分钟,再唤醒,就不会黑屏了,直到你重启,这个,tctien342 的文档里面有说的很清楚,你可以看下的

好嘞。我去看看去。我还在等这个的更新(*^▽^*)，希望能修复HDMI

**Comment by NominationP** [comment]
同样的问题，希望可以修复

**Comment by NominationP** [comment]
> 昨天尝试了tctien342的180421的引导后，发现Type-C转HDMI已经恢复正常了，但是原生HDMI还是闪屏，而且每次插上只能显示1秒就黑屏

请问一下，换引导的话要重新安装整个系统吗？从头开始？

**Comment by mxm646690173** [comment]
> > 昨天尝试了tctien342的180421的引导后，发现Type-C转HDMI已经恢复正常了，但是原生HDMI还是闪屏，而且每次插上只能显示1秒就黑屏
> 
> 请问一下，换引导的话要重新安装整个系统吗？从头开始？

不用，直接替换 efi 文件夹就行了
